### ISSUE TITLE
LightDock fails on "module 'pyparsing' has no attribute 'operatorPrecedence'"

### ISSUE THREAD

**Issue by brianjimenez** [lightdock/lightdock#20]
This is a problem reported at ProDy repository: https://github.com/prody/ProDy/issues/1417

Workaround is to reinstall ProDy from GitHub repository:

```bash
pip uninstall prody
pip install git+https://github.com/prody/ProDy.git
```

**Comment by brianjimenez** [comment]
Issue seems to have been fixed at ProDy side :)

**Comment by jamesmkrieger** [comment]
> Issue seems to have been fixed at ProDy side :)

Only using the workaround of installing from github. We still need to make a release to be able to use it from pypi.

**Comment by brianjimenez** [comment]
Thanks for the feedback @jamesmkrieger , will keep this issue open until there is a new release on PyPI.

**Comment by jamesmkrieger** [comment]
Ok

**Comment by jamesmkrieger** [comment]
Just to update you: I have now released ProDy 2.0.1, which includes this fix.
